docs/01-Headless/03-Directives.md

```diff
@@ -1,5 +1,244 @@
 # Directives
 
-## TODO
+## Overview
 
-To be done, issue #401
+Directives in AgnosUI are inspired by the **actions** in Svelte. These directives are element-level lifecycle functions that are executed when the element is created. They are updated (if an update function is provided) when a parameter changes, and finally, they are executed when the element is destroyed.
+These directives are typically used to add custom event handlers. For instance, core services like [focus track](../services/01-Focustrack.md) and [floating ui](../services/03-Floating-UI.md) create directives that are utilized by components such as [Select](../components/select/examples).
```

Comment:
If you want the relative links to work in GitHub, you would have to update from `../services/01-Focustrack.md` to `../04-Services/01-Focustrack.md`